FirebaseSharedSwift/Tests/third_party/DataEncoderTests.swift

```diff
@@ -482,33 +482,33 @@ class TestFirebaseDataEncoder: XCTestCase {
                    dataDecodingStrategy: .custom(decode))
   }
 
-  func testDecodingBase64StringAsBlobData() {
-    let data = "abcdef".data(using: .utf8)!
+  func testDecodingBase64StringAsBlobData() throws {
+    // Tests implicit migration of data that was written with .base64 (String type) using
```

Comment:
Nit: function comment should be above the function.

Reply:
Done. Thanks!